package.json

```diff
@@ -31,14 +31,11 @@
     "ajv": "^8.11.0",
     "antlr4": "4.8.0",
     "antlr4ts": "^0.5.0-alpha.4",
-    "cypress-multi-reporters": "^1.6.3",
-    "cypress-parallel": "^0.13.0",
     "json5": "^2.2.3",
     "mime": "^3.0.0",
     "mocha": "10.1.0",
     "performance-now": "^2.1.0",
     "plotly.js-dist": "^2.29.1",
-    "postinstall": "^0.7.4",
```

Comment:
Is postinstall not needed or is it part of core?

Reply:
postinstall was previously added for generating anltr artifacts after installing dependecies, but we later due to build issue start generating file manually, hence not need it anymore